src/graphql/data/ConversionRate.ts

```diff
@@ -0,0 +1,29 @@
+import { SupportedLocalCurrency } from 'constants/localCurrencies'
+import gql from 'graphql-tag'
+import ms from 'ms'
+import { getFetchPolicyForKey } from 'utils/getFetchPolicyForKey'
+
+import { useConvertQuery } from './__generated__/types-and-hooks'
+
+gql`
+  query Convert($toCurrency: Currency!) {
+    convert(fromAmount: { currency: USD, value: 1.0 }, toCurrency: $toCurrency) {
+      id
+      value
+      currency
+    }
+  }
+`
+
+export function useLocalCurrencyConversionRate(localCurrency: SupportedLocalCurrency, skip?: boolean) {
+  const { data, loading } = useConvertQuery({
+    variables: { toCurrency: localCurrency },
+    fetchPolicy: getFetchPolicyForKey(`convert-${localCurrency}`, ms('5m')),
```

Comment:
it was decided in the design doc: https://www.notion.so/uniswaplabs/Currency-Conversion-4b0c301fd52841e1a7b573c8a0a8293f

since the conversion rate isn't that crucial for trading information we decided that 5 minutes is good enough